lib/compat/wordpress-5.8.1/index.php

```diff
@@ -1,9 +0,0 @@
-<?php
-/**
- * Temporary compatibility shims for features present in Gutenberg.
- *
- * @package gutenberg
- */
-
-// Load the polyfills.
-require_once __DIR__ . '/widget-render-api-endpoint/index.php';
```

Comment:
Oh, sorry. This ended up landing in WP 5.9 not WP 5.8.1. You'll need to restore the `widget-render-api-endpoint` directory and move it to `wordpress-5.9.`.

Reply:
Done in ae7f42f.